docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/IndicateMissingValues.cs

```diff
@@ -1,32 +1,31 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Linq;
+using Microsoft.ML;
 using Microsoft.ML.Data;
 
-namespace Microsoft.ML.Samples.Dynamic
+namespace Samples.Dynamic
 {
     public static class IndicateMissingValues
     {
-
         public static void Example()
         {
             // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
             // as well as the source of randomness.
             var mlContext = new MLContext();
 
+            // Get a small dataset as an IEnumerable and convert it to an IDataView.
             var samples = new List<DataPoint>()
             {
-                new DataPoint(){ Label = 3, Features = new float[3] {1, 1, 0} },
-                new DataPoint(){ Label = 32, Features = new float[3] {0, float.NaN, 1} },
-                new DataPoint(){ Label = float.NaN, Features = new float[3] {-1, float.NaN, -3} },
+                new DataPoint(){ Features = new float[3] {1, 1, 0} },
+                new DataPoint(){ Features = new float[3] {0, float.NaN, 1} },
+                new DataPoint(){ Features = new float[3] {-1, float.NaN, -3} },
             };
-            // Convert training data to IDataView, the general data type used in ML.NET.
             var data = mlContext.Data.LoadFromEnumerable(samples);
 
-            // IndicateMissingValues is used to create a boolean containing
-            // 'true' where the value in the input column is NaN. This value can be used
-            // to replace missing values with other values.
-            IEstimator<ITransformer> pipeline = mlContext.Transforms.IndicateMissingValues("MissingIndicator", "Features");
+            // IndicateMissingValues is used to create a boolean containing 'true' where the value in the 
+            // input column is missing. For floats, missing values are float.NaN.
+            // The newly created value can be used to replace missing values with other values.
```

Comment:
>// The newly created value can be used to replace missing values with other values. [](start = 12, length = 83)

Indicator isn't used to replace. It's used to give extra information to the learner.